apps/studio/src/components/Modals/Generator/TemplateParameters.tsx

```diff
@@ -106,7 +106,7 @@ export const TemplateParametersSans: ForwardRefRenderFunction<TemplateParameters
     const servers = document?.servers();
     const availableServers: string[] = [];
     Object.entries(servers || {}).forEach(([serverName, server]) => {
-      if (supportedProtocols.includes(server.protocol())) availableServers.push(serverName);
+      if (server.protocol && supportedProtocols.includes(server.protocol())) availableServers.push(serverName);
```

Comment:
How can it be that `protocol` is not a function? What is it instead? 🤔

Reply:
So, based on what I noticed during the iteration, some index of server in `server.protocol` is returning `undefined`